index.md

```diff
@@ -1,131 +1,198 @@
+increase the divide between the rich and poor, leading to greater desperation and poverty which will drive 
+up crime.  
 
-Instead of bailing out corporations, who will not look out for citizens, governments should bail out the citizens themselves. Citizens will have the power to restart their life and will choose where to spend the government support money. This in turn will benefit the companies that the citizens need the most and the citizens of the country will decide which companies should survive and which ones should be left behind. 
+Instead of bailing out corporations, who will not look out for citizens, governments should bail out the 
+citizens themselves. Citizens will have the power to restart their life and will choose where to spend 
+the government support money. This in turn will benefit the companies that the citizens need the most and 
+the citizens of the country will decide which companies should survive and which ones should be left behind. 
 
 
 Focus on climate change and our environment
 -------------------------------------------
 
-The coronavirus pandemic has also shown how bad pollution is in this world. The most polluted cities suddenly have blue skies now that drastically fewer cars and planes are polluting our environment. It also showed that people who chronically live with bad air are much more likely to be negatively impacted by the coronavirus.
+The coronavirus pandemic has also shown how bad pollution is in this world. The most polluted cities suddenly 
+have blue skies now that drastically fewer cars and planes are polluting our environment. It also showed that 
+people who chronically live with bad air are much more likely to be negatively impacted by the coronavirus.
 
-We can take the lessons learned during the pandemic and reduce our impact on our planet? How can we commute less? Can we have more online meetings and travel less to business meetings? What sustainable technologies can we invest in to displace the polluting technologies?
+We can take the lessons learned during the pandemic and reduce our impact on our planet? How can we commute less? 
+Can we have more online meetings and travel less to business meetings? What sustainable technologies can we 
+invest in to displace the polluting technologies?
 
-Now that many sectors of greenhouse gas emitting industries are idled, we’ve drastically reduced our global carbon footprint. We must work hard to keep our CO2 emissions as low as possible as we return to a post Coronavirus world. We must protect our air, our oceans, our forests and our topsoil -- we can no longer steal from mother nature. We must invest in sustainable solutions so that we can care for mother nature. It is our only choice if we wish to survive as a species.
+Now that many sectors of greenhouse gas emitting industries are idled, we’ve drastically reduced our global 
+carbon footprint. We must work hard to keep our CO2 emissions as low as possible as we return to a post 
```

Comment:
I would not focus on CO2 only, and it's too technical. What about "to keep our polluting emissions as low as possible" ?